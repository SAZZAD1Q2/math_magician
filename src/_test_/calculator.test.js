import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from '../components/calculator';

describe('Calculator', () => {
  test('should render the component', () => {
    const component = renderer.create(<Calculator />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
