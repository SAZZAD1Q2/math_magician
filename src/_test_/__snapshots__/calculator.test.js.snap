// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator should render the component 1`] = `
<div
  className="calculator"
>
  <div
    className="left-container"
  >
    <div
      className="title-container"
    >
      <span
        className="title"
      >
        Let's do some math!
      </span>
    </div>
  </div>
  <div
    className="right-container"
  >
    <div
      className="calculator-content"
    >
      <div
        className="calculator-buttons"
      >
        <div
          className="screen"
        >
          <div
            className="display"
          >
            0
          </div>
        </div>
        <div
          className="buttons"
        >
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            AC
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            +/-
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            %
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
          >
            รท
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            7
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            8
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            9
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
          >
            x
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            4
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            5
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            6
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
          >
            -
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            1
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            2
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            3
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
          >
            +
          </button>
          <button
            className="number zeroDigitButton"
            onClick={[Function]}
            type="button"
          >
            0
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
          >
            .
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
